examples/teleop-elm/src/index.js

```diff
@@ -41,6 +41,9 @@ function onTrack(event) {
     } else {
       mediaElement.controls = true;
     }
+
+    const child = streamContainer.querySelector('video');
+    child?.remove();
```

Comment:
this makes it so new video feeds replace old ones, instead of being stacked below

Reply:
can you shorten to `streamContainer.querySelector('video')?.remove()`?

Reply:
> instead of being stacked below

Can you explain what this means and why it is not a good thing?

Reply:
> Can you explain what this means and why it is not a good thing?

Discussed IRL - the old code would result in a stale video element on top of a working video element - we want to remove the stale video element before inserting the new working one.